lib/tlscontext.c

```diff
@@ -236,6 +236,27 @@ tls_session_set_verify(TLSSession *self, TLSSessionVerifyFunc verify_func, gpoin
   self->verify_data_destroy = verify_destroy;
 }
 
+void
+tls_session_info_callback(const SSL *ssl, int where, int ret)
+{
+  TLSSession *self = (TLSSession*)SSL_get_app_data(ssl);
+  if( !self->peer_info_found && where == (SSL_ST_ACCEPT|SSL_CB_LOOP) ) {
+    X509* cert = SSL_get_peer_certificate(ssl);
```

Comment:
please try to use our coding conventions with white spaces. There's even PR #1180 where astyle can help you reformat your patches.